src/shared/dlt_offline_trace.c

```diff
@@ -149,7 +149,7 @@ void dlt_offline_trace_file_name(char *log_file_name, char *name, unsigned int i
     memset(log_file_name, 0, DLT_OFFLINETRACE_FILENAME_MAX_SIZE * sizeof(char));
     strncat(log_file_name, name, sizeof(DLT_OFFLINETRACE_FILENAME_BASE));
     strncat(log_file_name, DLT_OFFLINETRACE_FILENAME_DELI, sizeof(DLT_OFFLINETRACE_FILENAME_DELI));
```

Comment:
This is also size of the source.

Reply:
@votrungchi  
clang doesn't warn that line of code. 
But I changed to use the macro DLT_OFFLINETRACE_INDEX_MAX_SIZE and there also no warning anymore.

Reply:
Well, because of clang does not know this actually the size of source - I think it only warns when we use sizeof(source). It's would be good if we recalculate the remaining sizeof log_file_name (like you did at next line) after every strncat(). 
Best Regards.

Reply:
Ok. changed as your suggestion.